brew.py

```diff
@@ -61,9 +61,9 @@ def _invoke_shell_command(self, cmd: str, defaults: Mapping[str, Any]) -> int:
                 cmd,
                 shell=True,
                 cwd=self._context.base_directory(),
-                stdin=devnull if defaults["stdin"] else None,
-                stdout=devnull if defaults["stdout"] else None,
-                stderr=devnull if defaults["stderr"] else None,
+                stdin=devnull if !defaults["stdin"] else None,
+                stdout=devnull if !defaults["stdout"] else None,
+                stderr=devnull if !defaults["stderr"] else None,
```

Comment:
! is not a valid negation syntax in Python. It must be `not defaults[...].`

Reply:
Good call! I guess it's obvious I'm not a true Pythonista, hah. I'll update.